docker-compose.yml

```diff
@@ -0,0 +1,32 @@
+version: '3'
+services:
+  # chat:
+    # restart: always
+    # build:
+    #   context: ./chat_server/ 
+    #   dockerfile: chat_server.Dockerfile
+    # links:
+    #   - db:db
+    # env_file:
+    #    - ./chat_server/db.env
+    # ports:
+    #   - 3000:3000
```

Comment:
Should this section be activated ? If not maybe some clean up is possible. 
Anyway, I don't think the port is up to date (compared to `index.ts`)

Reply:
I removed all changes related to the docker branch 
This branch could be use to deploy in the future, but it is not yet completely functional